### ISSUE TITLE
const

### ISSUE THREAD

**Issue by olly-larkin** [LangProc/langproc-2018-cw#20]
Is the const keyword included in the spec?

**Comment by johnwickerson** [comment]
No. https://github.com/LangProc/langproc-2018-cw/commit/da377364b328e891361cfb49547fddbc475b9907
